### PR TITLE
Support multiple fields in `sort`

### PR DESCRIPTION
**Pull request by dominiklohmann**
This is a rather trivial implementation that's certainly not optimal in terms of performance, but I'd argue this is better than not having it.

### CODE REVIEW

**Comment by mavam** [comment]
How does that interplay with the `asc` and `desc` modifiers? Are they per field or globally?

**Comment by dominiklohmann** [comment]
> How does that interplay with the `asc` and `desc` modifiers? Are they per field or globally?

Per field, that's the only thing that really makes sense.

File: web/docs/operators/sort.md [hunk 1]
```diff
@@ -64,3 +68,9 @@ Arrange by field `foo` in descending order and put null values first:
 ```
 sort foo desc nulls-first
 ```
+
+Sort by the field `src_ip` and for matching values sort by `dest_ip`.
+
+```
+sort src_ip, dest_ip
```
Comment (tobim) [inline web/docs/operators/sort.md:75]: So if I want to specify the order I would write something like:
```
sort src_ip asc, src_bytes desc 
```
?

**Review by tobim — APPROVED** [review]
Looks pretty straight-forward.

File: web/docs/operators/sort.md (hunk 1 above)
Comment (dominiklohmann) [reply web/docs/operators/sort.md:75]: Yes. Let me add an example.

**Pull request merged**